src/artm/core/protobuf_helpers.h

```diff
@@ -49,6 +50,38 @@ void repeated_field_append(T* field, int index, V value) {
   field->Set(index, new_value);
 }
 
+template<class T>
+std::vector<bool> is_member(T a, T b) {
+  std::vector<bool> retval;
+
+  if (a.size() > 0) {
+    for (int i = 0; i < b.size(); ++i)
```

Comment:
From what I see you verify whether elements of "b" are present in "a" array. Matlab version works the other way around. I suggest to replace the order of you arguments to be consistent with Matlab's ismember.